### PR TITLE
feat(forms-web-app): add upload planning application form

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

UCD-1050

## Description of change

add upload planning application form

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [x] I have merged commits to avoid fixing things in this PR
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: forms-web-app/src/views/appellant-submission/upload-application.njk [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <div class="govuk-form-group">
+          <label class="govuk-heading-m" for="upload-application">
+            Upload a file
+          </label>
+          {% if errors['upload-application'] %}
+            <span id="upload-application-error" class="govuk-error-message">
+            <span class="govuk-visually-hidden">Error:</span> {{ errors['upload-application'].msg }}
+          </span>
+          {% endif %}
+          <input class="govuk-file-upload" id="upload-application" name="upload-application" type="file">
```
Comment (Fenisay) [inline forms-web-app/src/views/appellant-submission/upload-application.njk:50]: Just curiosity, what was wrong with the govuk input file ?

**Review by Fenisay — APPROVED** [review]
So there will be a screen that will validate the upload or not ?

File: forms-web-app/src/views/appellant-submission/upload-application.njk (hunk 1 above)
Comment (a6software) [reply forms-web-app/src/views/appellant-submission/upload-application.njk:50]: Right yeah, good spot. 

Nothing is wrong with it. 

The mistake I have made is that with the previous PR, the prototype called for a very specific CSS class being added to the label for the file upload component. That class conflicted with the one that is mandatory on the gov uk file upload nunjucks template. 

What I've done here is copy / paste from the other template, and in this instance, I don't need the class so should have used the component. Good spot.

Comment (a6software) [reply forms-web-app/src/views/appellant-submission/upload-application.njk:50]: Will do this as a separate PR

Comment (a6software) [reply forms-web-app/src/views/appellant-submission/upload-application.njk:50]: Disregard - this page has the exact same issue - they want a custom style on the "upload a file" text which isn't possible with nunjucks

Screenshot from 2020-11-25 08-17-06

**Pull request merged**